README.md

```diff
@@ -23,15 +23,10 @@ make install
 # Hermes Relayer
 # [Hermes](https://hermes.informal.systems/) is a Rust implementation of a relayer for the [Inter-Blockchain Communication (IBC)](https://ibcprotocol.org/) protocol.
 #
-# In order to use the hermes relayer you will need to check out a specific branch that can be used with interchain-accounts. 
+# Currently supported by Hermes v0.9.0
```

Comment:
Hermes v0.10.0 contains [conditional code](https://github.com/informalsystems/ibc-rs/blob/v0.10.0/relayer/src/channel/version.rs#L57-L62) which attempts to call the now removed endpoint `NegotiateAppVersion` if available, otherwise it fallbacks to logic to choose version based on a PortPrefix which is now incorrect for controller ports (ics27-owner) as it follows the [ics-27-1.](https://github.com/informalsystems/ibc-rs/blob/v0.10.0/modules/src/applications/mod.rs#L13) prefix.

Using hermes v0.9.0 everything works as expected.